midend/lib/Conversion/MatMulOptimization/MatMulParallelVectorization.cpp

```diff
@@ -0,0 +1,370 @@
+//===- MatMulParallelVectorization.cpp
```

Comment:
Wrong format.

Reply:
clang-format will automatically format the code like this.